README.md

```diff
@@ -1 +1,24 @@
-# kubebuilder-plugin
\ No newline at end of file
+# Enable kubebuilder-plugin for operator-sdk
+
+Update for kubebuilder-plugin path in go-mod pointing to the latest release.
+
+Install of Operator-SDK
+```
+go mod tidy
+make install
+```
+
+Run Operator init command to generate the sample java operator
+- Use the quarkus plugin flag
+- Pick domain name and project name as prefered.
+
+```
+operator-sdk init --plugins quarkus --domain xyz.com --project-name java-op
+```
+
+Once the operator is scaffolded check for the following files
+
+```
+Dockerfile	PROJECT		pom.xml	    src/main/java/com/xyz/JavaOpOperator.java 
+
+```
```

Comment:
I would run `tree` in the project and paste that output into the document instead.